hapi-fhir-jpaserver-model/src/main/java/ca/uhn/fhir/jpa/model/entity/ResourceTable.java

```diff
@@ -78,6 +78,7 @@
 	// Do not reuse previously used index name: IDX_INDEXSTATUS, IDX_RES_TYPE
 	@Index(name = "IDX_RES_DATE", columnList = BaseHasResource.RES_UPDATED),
 	@Index(name = "IDX_RES_TYPE_DEL_UPDATED", columnList = "RES_TYPE,RES_DELETED_AT,RES_UPDATED,PARTITION_ID,RES_ID"),
+	@Index(name = "IDX_RESID_UPDATED", columnList = "RES_ID,RES_UPDATED,PARTITION_ID"),
```

Comment:
Minor: It's nice if they all have the same prefix.
```suggestion
	@Index(name = "IDX_RES_RESID_UPDATED", columnList = "RES_ID,RES_UPDATED,PARTITION_ID"),
```